src/WarcraftLegacies.Source/Quests/Frostwolf/QuestSeaWitch.cs

```diff
@@ -41,7 +41,7 @@ public sealed class QuestSeaWitch : QuestData
         .AddAction(() =>
         {
           var triggerUnit = GetTriggerUnit();
-          if (GetOwningPlayer(triggerUnit) != FrostwolfSetup.Frostwolf?.Player) return;
+          if (triggerUnit != LegendFrostwolf.LegendThrall.Unit) return;
```

Comment:
I think the previous condition is correct - seems to reflect what's in the old GUI version.
![image](https://user-images.githubusercontent.com/8995690/204493729-9501a9be-93d3-4de4-8994-07681b8dcbb4.png)

Reply:
I just thought it'd make more sense thematically for the leader of the horde to recruit the units. We can revert if you want

Reply:
Yeah, I think we should revert and go with the old condition. If we use this new one we'd have to convey it to the player somehow. After all training Thrall in this scenario is not even mandatory.